components/Avatars/Avatar.css

```diff
@@ -0,0 +1,90 @@
+.header{
+  display: flex;
+   padding: 0.5rem 4rem;
+   justify-content: space-between;
+   align-items: center;
+   box-shadow: 2px 5px 8px grey;
+   
+ }
+ .Avatar_container{
+   width: 90%;
+   height: 10rem;
+   display: flex;
+   justify-content:center;
+   align-items: center;
+   border: 1px solid grey;
+   margin: 2rem 0;
+ }
+ 
+ .avatar-xl{
+   width:7em;
+   height:7em;
+   border-radius:50%;
+  
+ }
+ .avatar-lg{
+   width:5em;
+   height:5em;
+   border-radius:50%;
+  }
+ .avatar-std{
+   width:3em;
+   height:3em;
+    border-radius:50%;
+ }
+ .avatar-sm{
+   width:2em;
+   height:2em;
+    border-radius:50%;
+ }
+ .avatars{
+    float:left;
```

Comment:
can you tell why you used float 
I didn't get that part

Reply:
I used it initially, corrected it ,thanks for pointing out.